src/libs/functions/getHostname.ts

```diff
@@ -1,7 +1,22 @@
-export const getHostname = nativeEvent => {
+import { getErrorMessage } from 'cozy-intent'
+
+import { devlog } from '/core/tools/env'
+
+export const getHostname = (
+  nativeEvent?: { url?: string | unknown } | unknown
+): string | undefined => {
+  if (
+    !nativeEvent ||
+    typeof nativeEvent !== 'object' ||
+    !('url' in nativeEvent) ||
+    typeof nativeEvent.url !== 'string'
+  )
+    return
```

Comment:
Shouldn't we throw instead? Can this happen?

Reply:
I don't feel safe in throwing an error here as I don't know if the caller will expect it (fearing regressions). Logging a message instead then